### PR TITLE
feat: add design system freelance gig's first work package

### PR DESCRIPTION
**Pull request by rija**
This PR implements:

* a new freelance.html page with currently a description of:
  * the first work package for the design system
  * the first work package for the CMS deployment
* change to the index.html with a section for freelance gigs linking to the freelance page.

### CODE REVIEW

**Review by kencho51 — COMMENTED** [review]
Hi @rija,

**praise:** The new freelance page is working, and its link in main page is also working.

**polish:** Change `Market rates. As a package` to `Market rates as a package.`?

**question:** It mentions `Market rates. As a package`, but should we mention which market or region? Because `Market` is ambigous.

**Comment by rija** [comment]
@kencho51 fixed

File: jobs/tech/freelance.html [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+                    <dd>International market rates as a package</dd>
+                    <dt>How to apply:</dt>
+                    <dd>Send resume and/or portfolio as appropriate to tech AT gigasciencejournal.com</dd>
+                </dl>
+            </aside>
+        </section>
+        <section>
+            <h3>CMS: Setup CraftCMS instance </h3>
+
+            <p>Setup and instance of CraftCMS with following characteristics:</p>
```
Comment (pli888) [inline jobs/tech/freelance.html:78]: Should be `Setup an instance of CraftCMS`?

**Review by pli888 — APPROVED** [review]
**praise:** Looks good to me. Just correct a typo.

**Pull request merged**